mappings/net/minecraft/tag/GlobalTagAccessorRegistry.mapping

```diff
@@ -0,0 +1,18 @@
+CLASS net/minecraft/class_5413 net/minecraft/tag/GlobalTagAccessorRegistry
+	FIELD field_25741 TAG_ACCESSORS Ljava/util/Map;
+	METHOD method_30196 markReady ()V
+	METHOD method_30197 (Lnet/minecraft/class_5120;)Z
+		ARG 0 accessor
+	METHOD method_30198 apply (Lnet/minecraft/class_5415;)V
+		ARG 0 tagManager
+	METHOD method_30199 (Lnet/minecraft/class_5415;Lnet/minecraft/class_5120;)V
+		ARG 1 accessor
+	METHOD method_30200 (Lcom/google/common/collect/Multimap;Lnet/minecraft/class_5415;Lnet/minecraft/class_2960;Lnet/minecraft/class_5120;)V
+		ARG 2 id
+		ARG 3 accessor
+	METHOD method_30201 registerAccessor (Lnet/minecraft/class_2960;Ljava/util/function/Function;)Lnet/minecraft/class_5120;
+		ARG 0 id
+		ARG 1 managerGetter
+	METHOD method_30202 validateRegistrations ()V
+	METHOD method_30203 getDifferences (Lnet/minecraft/class_5415;)Lcom/google/common/collect/Multimap;
```

Comment:
Same as liach's comment here. It's `getMissingTags`. That's especially obvious if you look at the call hierarchy of this method.